index.html

```diff
@@ -89,27 +111,91 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    party: 'all'
   };
 
-  var svg;
+  var svg, x, y;
+
+  var margin = { top: 15, right: 15, bottom: 30, left:45 };
+
+  var width = 640 - margin.right -  margin.left;
+  var height = 400 - margin.top - margin.bottom;
 
   // FETCH DATA
 
-  function setup() {
+  d3.json('data.json', function (error, json){
+    // Call back function
+    if (error) { throw error; }
+
+    data = json;
+
+    setup();
+    update();
+  });
+
+  function setup( ) {
+
     // SVG
 
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform','translate(' + margin.left + ',' + margin.top + ')');
 
     // SCALES
 
+    x = d3.scale.linear()
+      .domain([d3.min(data,function (d){ return d.cfScore;}), d3.max(data, function (d){ return d.cfScore; } )])
+        .range([margin.left,(margin.left + width)])
+        .nice();
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d){ return d.distPart; } ))
+      .range([(height-margin.top),0])
+      .nice();    
+
     // AXES
 
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .tickSize(-height)
+      .orient("bottom");
+
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .tickSize(-width)
+      .orient("left");
+
+    svg.append("g")
+      .classed("x axis", true)
+      .attr("transform", "translate(0," + (height-margin.top) + ")")
+      .call(xAxis);
+
+    svg.append("g")
+      .classed("y axis", true)
+      .attr("transform","translate(" + margin.left + ",0)")
+      .call(yAxis);
+
+    var line = d3.svg.line()
+      .x(function(d) { return xScale(d.x); })
+      .y(function(d) { return yScale(d.y); });
+
+    svg.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate(0,"+(height/2)+")rotate(-90)")
```

Comment:
Note that `text` has `x` and `y` attributes that can be used instead of transform. These'll be nicer to work with as we won't have to do the string concatenation that is needed to create these `transform` values. (We'll still need `transform` for rotation, though.)